react/features/base/flags/constants.js

```diff
@@ -129,3 +129,9 @@ export const VIDEO_SHARE_BUTTON_ENABLED = 'video-share.enabled';
  * Default: disabled (false).
  */
 export const WELCOME_PAGE_ENABLED = 'welcomepage.enabled';
+
+/**
+ * Flag indicating if the Help button should be enabled.
+ * Default: enabled (true).
+ */
+export const HELP_BUTTON_ENABLED = 'help.enabled';
```

Comment:
Please sort this alphabetically.

Reply:
Hello ,  i am not able to disable help button in jitsi android sdk . i am using  "help.enabled" flag.

Reply:
This PR is not merged.

Reply:
how i can solve this PR ?

Reply:
hey @saghul i am facing the same issue. I want to hide or change help URL.

Please Guide.

Reply:
Please help us to solve this problem.
thanks